az-digital/az_quickstart
# Closes #1674 Fix up links in migrations

## Description
Links are broken in many migrations due to the use of tokens and urls that are not valid in Drupal 9.

## Related issues
Closes #1674

## How to test
Run a migration with Carousel Items
Check that `field_az_link` migrated correctly.

Run a migration with Events
Check that `field_az_link` migrated correctly
Check that `field_uaqs_location` migrated correctly

Run a migration with News
Check that `field_az_link` migrated correctly

Run a migration with Person
Check that `field_az_link` migrated correctly

Search indexing not  completing
Cron not completing
Sitemap generation not completing
This website has encountered an error 

## Types of changes

### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

### Drupal core
- Patch release changes
- Minor release changes

### Drupal contrib projects
- Patch release changes
- Minor release changes

## Checklist
